.github/workflows/main.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Did you forgot the 3.9 ? :)

Reply:
It was to make sure that #43 is not related with adding python3.9.